django_tenants/models.py

```diff
@@ -145,6 +147,16 @@ def get_primary_domain(self):
         except get_tenant_domain_model().DoesNotExist:
             return None
 
+    def reverse(self, request, view_name):
+        """
+        Returns the URL of this tenant.
+        """
+        url = 'https://' if request.is_secure() else 'http://'
```

Comment:
It's more then reverse is doing, I would suggest to leave just `//` and leave deciding the protocol by browser,  tought mail programs can have problem with such urls, as they don't have current used protocol. Also what if reverse is used in an email, so some notification sent by celery, without request? This shouldn't be looking at request, or request should be optional.